Sources/Subs-Db-postgresql.php

```diff
@@ -518,7 +523,7 @@ function smf_db_affected_rows($result = null)
  *
  * @param string $table The table (only used for Postgres)
  * @param string $field = null The specific field (not used here)
- * @param resource $connection = null The connection (if null, $db_connection is used)
+ * @param resource $connection = null The connection (if null, $db_connection is used) (not used here?)
```

Comment:
The mysql version can use it:
https://github.com/SimpleMachines/SMF2.1/blob/release-2.1/Sources/Subs-Db-mysql.php#L541